scripts/hid_configurator/configurator_core.py

```diff
@@ -677,19 +681,26 @@ def fetch_free_steps_buffor_info(dev, recipient):
 
     return struct.unpack(fmt, fetched_data)[0]
 
+
+interrupted = False
+
+
 def send_continuous_led_stream(dev, recipient):
-    while 1:
+    def signal_handler(signal, frame):
+        global interrupted
+        interrupted = True
+
+    signal.signal(signal.SIGINT, signal_handler)
+
+    while not interrupted:
         free_places = fetch_free_steps_buffor_info(dev, recipient)
         if free_places == None:
-            return None
+            continue
```

Comment:
Can we continue here, or we should return None ?

Reply:
Maybe add sleep here at least

Reply:
discussed offline